bin/dnsoterica

```diff
@@ -1,7 +1,6 @@
 #!/icg/bin/perl
 # ABSTRACT: run dns-o!
-package
-  dnsoterica;
+package dnsoterica;
```

Comment:
Had a _feeling_ it might've been something like that, but thought I'd highlight it anyway :-)